backend-api/src/functions/asyncCredential/sessionService/sessionService.ts

```diff
@@ -72,31 +73,39 @@ export class SessionService implements IGetActiveSession, ICreateSession {
   }
 
   async createSession(
-    sessionConfig: IAuthSession,
-  ): Promise<ErrorOrSuccess<null>> {
-    const config: IPutAuthSessionConfig = {
+    config: ICreateSessionConfig,
+  ): Promise<ErrorOrSuccess<string>> {
+    const sessionId = randomUUID();
+    const {
+      state,
+      sub,
+      client_id,
+      govuk_signin_journey_id,
+      redirect_uri,
+      issuer,
+    } = config;
+
+    const putItemCommandConfig: IPutAuthSessionConfig = {
```

Comment:
naming here isn't consistent with the interface

Reply:
updated